samples/grids/data-grid/load-save-layout/src/index.tsx

```diff
@@ -0,0 +1,378 @@
+                                  Save     
+                    </Button>
+                </div>
+
+                 <IgrDataGridToolbar
+                    ref={this.onToolbarRef}
+                    toolbarTitle="Sales Team"
+                    columnChooser="true"
+                    columnPinning="true"
+                />
+                
+                <IgrDataGrid
+                    ref={this.onGridRef}
+                    height="calc(100% - 2.75rem)"
+                    width="100%"
+                    rowHeight="50"
+                    autoGenerateColumns="false"
+                    dataSource={this.data}
+                    defaultColumnMinWidth="100"
+                    summaryScope="Root"
+                    isColumnOptionsEnabled="true"
+                    isGroupCollapsable="true"
+                    groupHeaderDisplayMode="Combined"
+                    groupSummaryDisplayMode="RowBottom"
+                    columnMovingMode="Deferred"
+                    columnMovingAnimationMode="SlideOver"
+                    columnMovingSeparatorWidth="2"
+                    columnShowingAnimationMode="slideFromRightAndFadeIn"
+                    columnHidingAnimationMode="slideToRightAndFadeOut"
+                    selectionMode="SingleRow"
+                    cornerRadiusTopLeft="0"
+                    cornerRadiusTopRight="0"
+                    >
+                    <IgrImageColumn field="Photo" headerText="Photo" contentOpacity="1"
+                    horizontalAlignment="stretch" width="110" paddingTop="5" paddingBottom="5"  paddingRight="10"/>
+                    <IgrTextColumn field="Name" width="*>140"/>
+
+                    <IgrTemplateColumn field="Sales" headerText="Sales" horizontalAlignment="center"
+                    paddingLeft="10" paddingRight="0"
+                        cellUpdating={this.onSalesCellUpdating} width="*>160" />
+
+                    <IgrTemplateColumn field="Productivity" headerText="Productivity"
+                    horizontalAlignment="center" width="*>150"
+                    template={this.getProductivityChart} />
+
+                    <IgrNumericColumn field="Salary" positivePrefix="$"
+                    showGroupingSeparator="true" width="*>160"/>
+
+                    <IgrDateTimeColumn field="Birthday" headerText="Date of Birth"
+                    horizontalAlignment="stretch" width="*>160" paddingRight="10"/>
+                    <IgrImageColumn field="CountryFlag" headerText="Country" contentOpacity="1"
+                    horizontalAlignment="stretch" width="130" paddingTop="5" paddingBottom="5" />
+
+                    <IgrTemplateColumn field="Address" headerText="Address" horizontalAlignment="left"
+                        cellUpdating={this.onAddressCellUpdating} width="*>160" />
+
+                    <IgrTemplateColumn field="Phone" horizontalAlignment="center"
+                        cellUpdating={this.onPhoneCellUpdating} width="140" />
```

Comment:
please replace **all templated columns** with Text/Numeric columns since they add just complexity to the sample and they are not needed to showcase load/save layout feature

Reply:
done

https://github.com/IgniteUI/igniteui-react-examples/pull/101/commits/af54a47b9c02875025df7f83cf81dc8e6813eb85